crates/networking/p2p/rlpx/eth/blocks.rs

```diff
@@ -111,10 +112,14 @@ impl GetBlockHeaders {
         } else {
             (self.skip + 1) as i64
         };
-        let limit = if self.limit > BLOCK_HEADER_LIMIT {
-            BLOCK_HEADER_LIMIT
+        let limit = if !cfg!(sync_test) {
+            if self.limit > BLOCK_HEADER_LIMIT {
+                BLOCK_HEADER_LIMIT
+            } else {
+                self.limit
+            }
         } else {
-            self.limit
+            env::var("SYNC-BATCH-SIZE").as_deref().unwrap_or("1024").parse().expect("Error parsing environmental variable")
```

Comment:
I think we could refactor this so we can use `BLOCK_HEADER_LIMIT` instead of parsing `"1024"`